style.css

```diff
@@ -0,0 +1,59 @@
+html {
+  font-size: 10px;
+}
+
+body {
+  font-size: 1.6rem;
+  margin: 0;
+  padding: 0;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+  height: auto;
+  margin: 0 auto;
+}
+
+.titlediv {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  /* background: lightgray; */
+}
+
+#title {
+  font-size: 46px;
+}
+
+#main {
+  border-radius: 5px;
+  background-color: #eee;
+  margin: 10px;
+  margin-top: 30px;
+}
+
+#img-div {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#img-caption {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+#tribute-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-left: 100px;
+  padding-right: 100px;
```

Comment:
It will be great to use padding propery insted of padding-left, right . It will reduce the additional memory usage.